flask/app/participants.py

```diff
@@ -57,19 +81,6 @@ def list_participants():
 
     columns = models.Participant.__table__.columns.keys()
 
-    raw_filters = request.args.getlist("filter")
-    if raw_filters:
-        filt = filter_query(models.Participant, raw_filters)
-        if type(filt) == str:
-            return filt, 400
-    else:
-        filt = [
-            (
-                models.Participant.participant_codename
-                == models.Participant.participant_codename
-            )
-        ]
-
     try:
         order_column = getattr(models.Participant, order_by_col)
```

Comment:
We should be able to order by `family_codename` as well. Not sure if we should allow ordering by `dataset_types` because I'm not sure what the expected behaviour should be.